src/SongPosition.h

```diff
@@ -11,41 +11,37 @@ struct lua_State;
 class SongPosition
 {
 public:
-	// Arcade - the current stage (one song).
-	// Oni/Endless - a single song in a course.
-	// Let a lot of classes access this info here so they don't have to keep their own copies.
-	float m_fMusicSeconds;		// time into the current song, not scaled by music rate
-	float m_fSongBeat;
-	float m_fSongBeatNoOffset;
-	float m_fCurBPS;
-	float m_fLightSongBeat;		// g_fLightsFalloffSeconds ahead
-	//bool m_bStop;			// in the middle of a stop (freeze or delay)
-	bool m_bFreeze;			// A flag to determine if we're in the middle of a freeze/stop.
-	bool m_bDelay;			// A flag to determine if we're in the middle of a delay (Pump style stop).
-	int m_iWarpBeginRow;		// The row used to start a warp.
-	float m_fWarpDestination;	// The beat to warp to afterwards.
-	RageTimer m_LastBeatUpdate;	// time of last m_fSongBeat, etc. update
-	float m_fMusicSecondsVisible;
-	float m_fSongBeatVisible;
+	SongPosition();
 
-	SongPosition()
-		: m_fMusicSeconds(0.0f),
-		  m_fSongBeat(0.0f),
-		  m_fSongBeatNoOffset(0.0f),
-		  m_fCurBPS(0.0f),
-		  m_fLightSongBeat(0.0f),
-		  //m_bStop(false),
-		  m_bFreeze(false),
-		  m_bDelay(false),
-		  m_iWarpBeginRow(0),
-		  m_fWarpDestination(0.0f),
-		  m_LastBeatUpdate(RageZeroTimer),
-		  m_fMusicSecondsVisible(0.0f),
-		  m_fSongBeatVisible(0.0f)
+	float m_fMusicSeconds; // Represents the time into the current song, not scaled by the music rate.
```

Comment:
Let's move the comments to being above the variable. they're kind of long and here the comments don't align